public/components/userpage.js

```diff
@@ -0,0 +1,100 @@
+
+ajaxRequest("GET", window.location + "?ajax=true", {}, {}, renderComponent);
```

Comment:
okok i thought that was the issue
i think its just a bit weird from a tech perspective, buuu to chrome for not working